tools/integration_tests/util/setup/setup.go

```diff
@@ -111,7 +110,7 @@ func CompareFileContents(t *testing.T, fileName string, fileContent string) {
 	}
 
 	if got := string(content); got != fileContent {
-		t.Errorf("File content doesn't match. Expected: %q, Actual: %q", fileContent, got)
```

Comment:
Can we not undo this fix and leave it as it is?

Reply:
what is undo without reverting the PR?